state/protocol/badger/mutator.go

```diff
@@ -867,14 +878,14 @@ func (m *FollowerState) epochMetricsAndEventsOnBlockFinalized(parentEpochState,
 		return
 	}
 	// Transition from Setup phase to Committed phase. `finalized` is first block in Committed phase.
-	if parentEpochPhase == flow.EpochPhaseSetup && childEpochPhase == flow.EpochPhaseCommitted {
+	if parentEpochPhase != flow.EpochPhaseCommitted && childEpochPhase == flow.EpochPhaseCommitted {
 		events = append(events, func() { m.metrics.CurrentEpochPhase(flow.EpochPhaseCommitted) })
 		events = append(events, func() { m.consumer.EpochCommittedPhaseStarted(childEpochCounter, finalized) })
 		return
 	}
 
-	// TODO(6013) efm-recovery: need to update metrics logic for EFM recovery
-	return nil, nil, nil
+	// TODO(6092) efm-recovery: need to decide how an EpochFallback phase would like like to re-enable below sanity check
```

Comment:
In other PR I am using `TODO(EFM, #6092)` that is easily searchable and GH detects it as link.